parm/config/config.fcst

```diff
@@ -251,7 +251,7 @@ elif [[ "$CDUMP" == "gfs" ]] ; then # GFS cycle specific parameters
     # Write gfs restart files to rerun fcst from any break point         
     export restart_interval_gfs=${restart_interval_gfs:-0}
     if [ $restart_interval_gfs -le 0 ]; then
-        export restart_interval=0
+        export restart_interval="$FHMAX_GFS"
```

Comment:
It's actually not. a restart_interval of 0... just writes a restart at the end. The substitution is needed to make the next bit work , because AFAIK having 0 be one of multiple restart_intervals is not the same as 0 by itself.

Reply:
Ah, right, thanks for explaining! :)